arrow-optics/src/main/kotlin/arrow/optics/Fold.kt

```diff
@@ -1,19 +1,6 @@
 package arrow.optics
 
-import arrow.Kind
-import arrow.core.Const
-import arrow.core.Either
-import arrow.core.ListK
-import arrow.core.None
-import arrow.core.Option
-import arrow.core.Some
-import arrow.core.extensions.AndMonoid
-import arrow.core.extensions.listk.monoid.monoid
-import arrow.core.extensions.monoid
-import arrow.core.identity
-import arrow.higherkind
-import arrow.typeclasses.Foldable
-import arrow.typeclasses.Monoid
+import arrow.core.*
```

Comment:
Not sure if ktlint is setup to allow this

Reply:
It doesn't. We don't allow star imports.

Reply:
This must be my IntelliJ config, I'll update the settings to use single name imports.